### PR TITLE
Reference management models content type v9

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
Belongs to #3145

### CODE REVIEW

**Comment by busrasengul** [comment]
You're on fire @erikjanwestendorp 
We'll check this tomorrow 💪🏼

File: Reference/Management/Models/ContentType-v9.md [hunk 1]
```diff
@@ -0,0 +1,357 @@
+---
+versionFrom: 9.0.0
+verified-against: rc001
+meta-title: ContentType Model
```
Comment (busrasengul) [inline Reference/Management/Models/ContentType-v9.md:4]:
```suggestion
meta.Title: "ContentType Model"
```

File: Reference/Management/Models/ContentType-v9.md [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+meta.Description: A ContentType corresponds to the Document Type found in the backoffice.
```
Comment (busrasengul) [inline Reference/Management/Models/ContentType-v9.md:5]:
```suggestion
meta.Description: "A ContentType corresponds to the Document Type found in the backoffice."
```

**Review by busrasengul — CHANGES_REQUESTED** [review]

File: Reference/Management/Models/ContentType-v9.md (hunk 2 above)
Comment (erikjanwestendorp) [reply Reference/Management/Models/ContentType-v9.md]: @busrasengul Updated 😄

**Pull request merged**